docs/jedis5-breaking.md

```diff
@@ -2,12 +2,34 @@
 
 - `StreamConsumersInfo` has been renamed to `StreamConsumerInfo`.
 
+- Following methods have been renamed:
+  - from `sdiffstore` to `sdiffStore`
+  - from `sinterstore` to `sinterStore`
+  - from `sintercard` to `sinterCard`
+  - from `sunionstore` to `sunionStore`
+  - from `zrangestore` to `zrangeStore`
+  - from `zinterstore` to `zinterStore`
+  - from `zintercard` to `zinterCard`
+  - from `zunionstore` to `zunionStore`
```

Comment:
I don’t quite agree with the break change brought about by the name change. `set` and `zset` are data structures that are widely used, **and there will not be too many updates and developments in the future**, maybe we should turn a blind eye and let this name exist.